cmd/dump.go

```diff
@@ -164,6 +170,16 @@ func runDump(ctx *cli.Context) error {
 	return nil
 }
 
+func getDumpFileName(ctx *cli.Context) string {
+	fmtString := ctx.String("file")
+
+	if strings.Count(fmtString, "%d") > 0 {
+		fmtString = fmt.Sprintf(fmtString, time.Now().Unix())
```

Comment:
In reference to my other comment, personally I think this is unnecessary.

If a user wants to override the file name, they should be able to provide the exact name they want, without relying on the code transforming a %d. This code would only transform a single %d as well, and if they had any other formatting symbols before/after, they would also end up with a bad filename.

I will not block this PR, however I don't personally approve of this "magic".
(To clarify, I do like this PR. Overriding the filename would be a great feature. I just don't particularly care for this part.

Reply:
Sorry for the "dupe" comment notification, I forget that comments in the file section don't show up in conversation unless it's marked as a review.

Reply:
i see your point and generally agree. but if you don't submit a custom file name with the `--file` argument, it'll use the default and the default contains `%d`. so, this check is necessary.
we could change the default to not include `%d` but that may break existing setups?

Reply:
Or change the default value to 
```go
Value: fmt.Sprintf("gitea-dump-%d.zip", time.Now().Unix())
```

Reply:
some pseudo code could be:

```
filename := fmt.Sprintf("gitea-dump-%d.zip", time.Now().Unix())
if ctx.String("file") {
    filename = ctx.String("file")
}
```
and don't do fmt.Sprintf on what is passed in via CLI

Reply:
> `Value: fmt.Sprintf("gitea-dump-%d.zip", time.Now().Unix())`

damn. that's brilliant. i'll do that. thank you, sir!

Reply:
Either that or what @techknowlogick said. Whichever seems cleaner in the code. :+1:

Reply:
implemented @jolheiser's approach.

Reply:
:tada: 
Thanks again for the PR!